test/CMakeLists.txt

```diff
@@ -255,6 +255,7 @@ add_sdl_test_executable(testcustomcursor SOURCES testcustomcursor.c)
 add_sdl_test_executable(testvulkan NO_C90 SOURCES testvulkan.c)
 add_sdl_test_executable(testoffscreen SOURCES testoffscreen.c)
 add_sdl_test_executable(testpopup SOURCES testpopup.c)
+add_sdl_test_executable(dialog SOURCES dialog.c)
```

Comment:
This should probably be called `testdialog` to match the other test programs

Reply:
Good idea, feel free to submit a PR for this.